common/compositor/gl/egloffscreencontext.cpp

```diff
@@ -102,22 +103,22 @@ void EGLOffScreenContext::RestoreState() {
 
 EGLint EGLOffScreenContext::GetSyncFD() {
   EGLint sync_fd = -1;
-#ifndef DISABLE_EXPLICIT_SYNC
-  EGLSyncKHR egl_sync =
-      eglCreateSyncKHR(egl_display_, EGL_SYNC_NATIVE_FENCE_ANDROID, NULL);
-  if (egl_sync == EGL_NO_SYNC_KHR) {
-    ETRACE("Failed to make sync object.");
-    return -1;
-  }
+  if (android::DrmHwcTwo::IsExplicitSyncEnabled()) {
+    EGLSyncKHR egl_sync =
```

Comment:
white space ? common/ should not include any android specific or linux specific headers